k8s/scripts/cis-template.jinja2

```diff
@@ -0,0 +1,34 @@
+<!-- markdownlint-disable -->
```

Comment:
I would rather work on making the text compliant:
 -  It is easier to fix here than after the md is generated
 - the mdlint failing can be a good indicator that we have generated garbage

Reply:
Done